### PR TITLE
Fix panic from multiple probe cleanup calls.

### PR DESCRIPTION
**Pull request by timstclair**
Fixes https://github.com/kubernetes/kubernetes/issues/20566

### CODE REVIEW

File: pkg/kubelet/prober/worker.go [hunk 1]
```diff
@@ -117,6 +117,14 @@ probeLoop:
 	}
 }
 
+// stop stops the probe worker. The worker handles cleanup and removes itself from its manager.
```
Comment (yujuhong) [inline pkg/kubelet/prober/worker.go]: nit: also comment here saying it's safe to to call stop multiple times.

**Comment by yujuhong** [comment]
LGTM with one nit. Thanks for the quick fix!

File: pkg/kubelet/prober/worker.go (hunk 1 above)
Comment (timstclair) [inline pkg/kubelet/prober/worker.go]: done.

**Comment by timstclair** [comment]
Rebased.

**Pull request merged**